Lib/zipapp.py

```diff
@@ -179,6 +195,12 @@ def main(args=None):
             help="Display the interpreter from the archive.")
     parser.add_argument('source',
             help="Source directory (or existing archive).")
+    parser.add_argument("--include", "-i", default=["*"], nargs="*",
+            help="A glob-style sequence of paths to include.",
+    )
+    parser.add_argument("--exclude", "-e", default=[], nargs="*",
+            help="A glob-style sequence of paths to exclude.",
+    )
```

Comment:
What are the use cases for these defaults? They seem reasonable, but personally I've never needed this functionality, so I have no easy way to judge whether this would be more useful than (for example) the ability to exclude a list of named subdirectories.

I'd expect the command line API to handle basic cases, but for more complex uses I'd assume people would write a custom build script that used the `create_archive` function directly. So I'd like to see some evidence that this particular form of filter is sufficiently common to warrant being a feature of the command line version - otherwise it could become a precedent for more and more complexity in the CLI, which is something I think we should avoid.